nixos/modules/services/hardware/keyd.nix

```diff
@@ -0,0 +1,68 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta = {
+    maintainers = [ "cidkid" ];
```

Comment:
Please add yourself to `maintainers/maintainer-list.nix` and then reference the entry from there.

Reply:
fixed